README.md

```diff
@@ -17,6 +17,11 @@ The project consists of two parts:
     <artifactId>spline-core</artifactId>
     <version>{{site.spline.version}}</version>
 </dependency>
+<dependency>
+    <groupId>za.co.absa.spline</groupId>
+    <artifactId>spline-core-spark-adapter-2.2</artifactId>
```

Comment:
Please add XML comment explaining what 2.2 means. It might not be evident to a user that 2.2 refers the spark version that _he_ is using.